<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmSliderRuler</class>
 <widget class="QWidget" name="frmSliderRuler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="SliderRuler" name="widget1" native="true"/>
   </item>
   <item>
    <widget class="SliderRuler" name="widget2" native="true"/>
   </item>
   <item>
    <widget class="SliderRuler" name="widget3" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SliderRuler</class>
   <extends>QWidget</extends>
   <header location="global">sliderruler.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
